src/storage/compression/compression.cpp

```diff
@@ -394,6 +394,8 @@ void IntegerBitpacking<T>::setValuesFromUncompressed(const uint8_t* srcBuffer, o
     auto chunkStart = (uint8_t*)getChunkStart(dstBuffer, posInDst, header.bitWidth);
     auto startPosInChunk = posInDst % CHUNK_SIZE;
     U chunk[CHUNK_SIZE];
+    // TODO(bmwinger): Now I'm a bit confused here. Why do we always need to unpack the chunk first?
+    //      Can't we just unpack the chunk if we need to modify only partial of it?
```

Comment:
Yes, I guess this should only be the behaviour for partial chunks, and we can avoid unpacking when writing whole chunks. This was originally written to work on individual values and I hadn't spent too much time optimizing it.

Reply:
OK. I will keep the TODO comment there.